### PR TITLE
Installation Documentation

### PR DESCRIPTION
**Pull request by havok2063**
This PR updates the `jdaviz` python package installation documentation and expands the section on running jdaviz.

### CODE REVIEW

**Comment by havok2063** [comment]
@javerbukh Working on it.  Apparently my repo is quite out of sync in strange ways.  Is there any way to rebase accepting all incoming changes on conflicts and let it sort itself out?  I went through a long manual rebase already but still have missing files, e.g. the `docs/logos`.  So I might start over.

**Comment by havok2063** [comment]
@javerbukh I think I've fixed my out-of-sync issues.

File: docs/installation.rst [hunk 1]
```diff
@@ -1,24 +1,65 @@
...
-Note that to load data in the notebook version, uncomment the cell in the notebook and
-provide the path to your data.
+or simply start a new Jupyter notebook and run the following in a cell::
+
+    from jdaviz import Application
+
+    app = Application()
+    app
+
+To learn more about the various ``jdaviz`` application configurations and loading data, see xxxx.
```
Comment (nmearl) [inline docs/installation.rst:65]: Where does this intend on pointing to?

Comment (havok2063) [reply docs/installation.rst]: I was intending to point this to whatever sort of Getting Started page got created, or an intro page for CubeViz, SpecViz, etc.  If we have a placeholder reference, I can pop that in.

File: docs/installation.rst [hunk 2]
```diff
@@ -1,24 +1,65 @@
...
+``jdaviz`` provides a command-line tool to start the web application. To see the syntax and usage,
+from a terminal, type::
+
+    jdaviz --help
 
-For the notebook version, start the jupyter kernel with the path to the Example.ipynb notebook::
+The general syntax to start the application is to provide a local filename path and an application configuration
+to load, i.e.::
+
+    jdaviz /path/to/data/file --layout=[configuration]
```
Comment (javerbukh) [inline docs/installation.rst:43]: I think `--layout=<configuration>` would make more sense here.

Comment (havok2063) [reply docs/installation.rst]: Done!

**Pull request merged**